config/examples/Creality/CR-10 V3/Configuration.h

```diff
@@ -76,7 +76,7 @@
 // @section info
 
 // Author info of this build printed to the host during boot and M115
-#define STRING_CONFIG_H_AUTHOR "Aaron Just, CR10-V3)" // Who made the changes.
+#define STRING_CONFIG_H_AUTHOR "PYRE" // Who made the changes.
```

Comment:
Changed to me !

Reply:
You shouldn't remove config author credits.

Reply:
Hi, sorry I was assuming I could with the comment. I would not have changed it with "Who create the config file" as comment.
I just changed it back to original in my repository.
Should I do an extra pull request for that or is there a way to modify this within this pull request ? I'm quite new on Github I'm still discovering

Reply:
I've my answer I think, it seems to be automatically applied to the pull request